go-controller/pkg/libovsdbops/portgroup.go

```diff
@@ -96,8 +137,11 @@ func AddPortsToPortGroupOps(nbClient libovsdbclient.Client, ops []libovsdb.Opera
 	return m.CreateOrUpdateOps(ops, opModel)
 }
 
-// AddPortsToPortGroup adds the provided ports to the provided port group
+// AddPortsToPortGroup adds the provided ports to the provided port group if they do not already exist
 func AddPortsToPortGroup(nbClient libovsdbclient.Client, name string, ports ...string) error {
+	if PortGroupHasPorts(nbClient, name, ports) {
```

Comment:
maybe move this condition to `AddPortsToPortGroupOps` and then we don't need to check the same in `handleLocalPodSelectorAddFunc`?

Reply:
Talking to Jaime he wants me to limit the scope of doing the cache lookup to only network policy and move the function out of libovsdbops and into a network policy file. So I will remove this call here since it is not used by network policy.